frontend/src/components/form/hash-input.jsx

```diff
@@ -0,0 +1,63 @@
+import { createEffect, createSignal } from "solid-js";
+import { styled } from "solid-styled-components";
+import icon from "../../assets/copy-ico.svg";
+
+const HashInput = (props) => {
+  const [value, setValue] = createSignal(
+    props.defaultValue || `djkbfsdjkbfsdj`
```

Comment:
Please, change `djkbfsdjkbfsdj` and use '' brackets when you do not need string interpolation

Reply:
done